frontend/lib/onboarding/onboarding-step-3.tsx

```diff
@@ -338,25 +343,44 @@ export default class OnboardingStep3 extends React.Component<
   }
 
   getSuccessRedirect(leaseType: string): string {
-    for (let info of this.leaseModals) {
-      if (info.leaseType === leaseType) {
-        return info.route;
+    const housingTypeFromSession = window.sessionStorage.getItem("housingType");
```

Comment:
When I reload while on this step 3 I get this error. Do you get the same thing?
![image](https://github.com/user-attachments/assets/592ff597-34ee-44da-bf1b-4a429b48ab15)
I see plenty of other places where `window` is referenced without any extra checks that it's defined, so I'm not sure what the issue is.

Reply:
hm yeah i'm getting the same error. i never refreshed on this page so didn't see this! will troubleshoot